<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HipotecarDialog</class>
 <widget class="QDialog" name="HipotecarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Hipotecar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPropiedades">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Propiedades:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Hipotecar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Deshipotecar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Propiedades Hipotecadas:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboHipotecadas">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitar Hipoteca</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Dinero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dineroJugador">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
